sklearn/feature_selection/rfe.py

```diff
@@ -65,6 +67,30 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         If within (0.0, 1.0), then ``step`` corresponds to the percentage
         (rounded down) of features to remove at each iteration.
 
+    tune_step_at : int or float or None, optional (default=None)
+        Number of remaining features reached when ``tuning_step`` is used
+        rather than ``step``. May be specified as an (integer) number of
+        remaining features or, if within (0.0, 1.0), a percentage (rounded
+        down) of the original number of features. If original number of
+        features and parameter settings would result in stepping past
+        ``tune_step_at``, then the number of features removed in the iteration
+        prior to stepping over will adjust to arrive at this value.
+
+    tuning_step : int or float, optional (default=1)
+        Step to use starting at ``tune_step_at`` number of remaining features.
+        If greater than or equal to 1, then ``tuning_step`` corresponds to the
+        (integer) number of features to remove at each iteration. If within
+        (0.0, 1.0), then ``tuning_step`` corresponds to the percentage (rounded
+        down) of features to remove at each iteration.
```

Comment:
of which features?

Reply:
The reason why its left out which features (original or remaining) is because it depends on reducing_step boolean setting.  Doc description for step parameter is same.